docs/README.md

```diff
@@ -0,0 +1,36 @@
+# Nebula docs
+---
+
+**Nebula** is a distributed, scalable, lightening-fast graph database. It is the only solution in the world capable to host graphs with dozens of billions of vertices (nodes) and trillions of edges, while still provides millisecond latency.
```

Comment:
I have a research this time. Seems that only one form is legal, i.e. _lightning-fast_, not lighting-fast, not lightenting-fast.

Reply:
thank you so much for that